src/main/kotlin/uk/gov/justice/hmpps/offendersearch/config/DocketConfiguration.kt

```diff
@@ -0,0 +1,56 @@
+package uk.gov.justice.hmpps.offendersearch.config
```

Comment:
This was in ResourceServerConfig which has now been deleted with its bits going into the security package